pulsar-client/src/main/java/org/apache/pulsar/client/api/ConsumerConfiguration.java

```diff
@@ -26,6 +26,9 @@
 import java.util.concurrent.TimeUnit;
 
 import org.apache.pulsar.client.impl.conf.ConsumerConfigurationData;
+import org.apache.pulsar.common.api.proto.PulsarApi.CommandSubscribe.InitialPosition;
+import org.apache.pulsar.common.api.proto.PulsarApi.CommandSubscribe.InitialPosition;
+import org.apache.pulsar.common.api.proto.PulsarApi.CommandSubscribe.InitialPosition;
```

Comment:
In any case, we should have a different enum for `InitialPosition` for the user facing API, rather that exposing the `PulsarApi` which is for wire-protocol.